src/controller/LoginController.php

```diff
@@ -2,43 +2,50 @@
 
 class LoginController {
 
+    const COOKIE_EXPIRE_TIME = 3600;
+
     private $connection;
 
     function __construct($connection) {
         $this->connection = $connection;
     }
 
     public function loginAction() {
-        if (isset($_POST['userName']) && isset($_POST['password'])) {
+
+        if (empty($_POST['userName']) && empty($_POST['password'])) {
+            include("views/login.php");
+        } else {
             $userName = $_POST['userName'];
             $password = $_POST['password'];
             $status = $this->userExist($userName, $password);
 
-            if ($status == 1) {
-                setcookie("login", 1, time() + 60, "/");
-                header("Location: index.php?controller=welcome&action=welcome");
-            } elseif ($status == 2) {
-                setcookie("error", "Klaida, prisijungti nepavyko ", time() + 15, "/");
+            if ($status == FALSE) {
+                include("views/error.php");
+            } elseif ($status == TRUE) {
+                setcookie("login", 'login', time() + self::COOKIE_EXPIRE_TIME, "/");
+                $_GET['controller'] = "welcome";
+                $_GET['action'] = "welcome";
+                header("Location: index.php");
             }
-        } else {
-            include("views/login.php");
         }
     }
 
     public function logoutAction() {
         if (isset($_POST['logout'])) {
-            setcookie("login", 1, time() - 60, "/");
-            header("Location: index.php?controller=login&action=login");
+            setcookie("login", 'logout', time() - self::COOKIE_EXPIRE_TIME, "/");
+            $_GET['controller'] = "login";
+            $_GET['action'] = "login";
+            header("Location: index.php");
```

Comment:
Galėčiau lažintis kad ištrynus 36 ir 37 eilutes logout funkcija veiktų taip pat gerai. Išsiaiškink kodėl ir papasakok man

Reply:
Lažybų tikrai neisiu, nes tu esi visiškai teisus, mes kaip praeitą kartą rašėm kodą, kai ateinį į index.php, be GET parametro reikšmių, index nukreipia į welcome controllerį, o šis pamatęs jog nėra COOKIE ["login"] suteikia login reikšmes GET parametram ir vėl perkelia į index.php , o index nukreipia į login controllerį, kuriame nukreipiama į prisijungimo formą.